apps/web/scripts/kernel/handlers/aptos.ts

```diff
@@ -9,54 +9,36 @@ type Payload = {
 	transaction?: string;
 };
 
-export const transferCoin: HandleMethod<Payload> = ({ payload }) => {
-	if (!payload.privateKey || !payload.transaction) {
-		throw Error('Missing privateKey or transaction');
-	}
-
-	const signatureString = aptosHandler.handleTransferCoin(
-		payload.privateKey,
-		payload.transaction,
-	);
-
-	respond(payload.requestId, ResponseCode.SUCCESS, { signatureString });
-};
-
-export const transferToken: HandleMethod<Payload> = ({ payload }) => {
-	if (!payload.privateKey || !payload.transaction) {
-		throw Error('Missing privateKey or transaction');
-	}
-
-	const signatureString = aptosHandler.handleTransferToken(
-		payload.privateKey,
-		payload.transaction,
-	);
-
-	respond(payload.requestId, ResponseCode.SUCCESS, { signatureString });
-};
-
-export const claimToken: HandleMethod<Payload> = ({ payload }) => {
-	if (!payload.privateKey || !payload.transaction) {
-		throw Error('Missing privateKey or transaction');
-	}
-
-	const signatureString = aptosHandler.handleClaimToken(
-		payload.privateKey,
-		payload.transaction,
-	);
-
-	respond(payload.requestId, ResponseCode.SUCCESS, { signatureString });
-};
-
-export const updateDirectTransfer: HandleMethod<Payload> = ({ payload }) => {
-	if (!payload.privateKey || !payload.transaction) {
-		throw Error('Missing privateKey or transaction');
-	}
-
-	const signatureString = aptosHandler.handleUpdateDirectTransfer(
-		payload.privateKey,
-		payload.transaction,
-	);
-
-	respond(payload.requestId, ResponseCode.SUCCESS, { signatureString });
-};
+type AptosHandle = (
+	privateKey: Uint8Array,
+	transaction: string,
+) => Promise<string>;
+
+export const constructAptosHandle =
```

Comment:
I think it's not a construction, it seems like `getAptosHandle` returns a handle with a passed callback. Otherwise, you can use just one handle and pass `type` to call corresponding utils